core/src/main/resources/lib/layout/copyButton/copyButton.js

```diff
@@ -7,28 +7,17 @@ Behaviour.specify(
       // HTMLUnit 2.70.0 does not recognize isSecureContext
       // https://issues.jenkins.io/browse/JENKINS-70895
       if (!window.isRunAsTest && isSecureContext) {
```

Comment:
```suggestion
      if (isSecureContext) {
```

I just found out that [HTMLUnit 3.1.0](https://htmlunit.sourceforge.io/changes-report.html#a3.1.0) added support for `isSecureContext`. Which version is currently used for testing? Maybe this can be removed like this

Reply:
We testing with HTMLUnit 3.5.0 per https://github.com/jenkinsci/jenkins-test-harness-htmlunit/releases/tag/154.v973ee207a_b_95